src/hubspot-provider.ts

```diff
@@ -0,0 +1,103 @@
+const Pkg = require('../package.json');
+const hubspot = require('@hubspot/api-client');
+
+type HubspotProviderOptions = {}
+
+function HubspotProvider(this: any, options : HubspotProviderOptions) {
+	const seneca:any = this;
+
+	const entityBuilder = this.export('provider/entityBuilder');
+
+	seneca.message('sys:provider,provider:hubspot,get:info', get_info)
+	async function get_info(this: any, _msg: any) {
+		return {
+			ok: true,
+			name: 'hubspot',
+			version: Pkg.version,
+			sdk: {
+				name: 'hubspot-node',
+				version: Pkg.dependencies['hubspot'],
+			}
+		}
+	}
+	entityBuilder(this, {
+		  provider: {
+			  name: 'hubspot'
+		  },
+		  entity: {
+			  company: {
+				  cmd: {
+					  list: {
+						  action: async function(this: any, entize: any, msg: any) {
+							  let res, list;
+							  res = (await this.shared.sdk.crm.companies.basicApi.getPage(10)).results;
+							  list = res.map((data) => entize(data))
+							  return list;
+						  }
+					  },
+					  load: {
+						  action: async function(this: any, entize: any, msg: any) {
+							  let data = (msg.q.id.split('/'));
+							  let id = data[0];
+							  let obj = {};
+							  data = data.slice(1);
+							  try{
```

Comment:
no need for try catch here - the function is async so the exception will be caught further up the call stack